### PR TITLE
added test to cover scenario when only cvv is displayed

### PR DESCRIPTION
**Pull request by msakowicz**
Before merge have  to change branch to develop
Waiting for: https://github.com/SecureTrading/js-payments/tree/feature/ST-357-allow-returning-customer-to-only-submit-securitycode

### CODE REVIEW

File: src/main/resources/__files/configSubmitCvvOnly.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "analytics": true,
+  "animatedCard": false,
+  "deferInit": false,
+  "fieldsToSubmit": ["code"],
```
Comment (james-simpson-ST) [inline src/main/resources/__files/configSubmitCvvOnly.json]: We probably want to make the fieldsToSubmit match the field names we have in the backend of these fields

This should be "pan", "expirydate" and "securitycode" 

Can you share this with Grzegorz and in this case fieldsToSubmit should be ["securitycode"]

File: src/test/java/com/SecureTrading/pageobjects/PaymentPage.java [hunk 2]
```diff
@@ -483,4 +484,28 @@ public void validateIfFieldHasCorrectStyle(FieldType fieldType, String expectedS
     public void waitUntilPageIsLoaded() throws InterruptedException {
         waitUntilElementIsDisplayed(visaCheckoutMockButton, 8);
     }
+
+    public void validateIfFieldIsPresent(FieldType fieldType) {
```
Comment (james-simpson-ST) [inline src/test/java/com/SecureTrading/pageobjects/PaymentPage.java]: Should this be called validateIfFieldIs**Not**Present

Comment (msakowicz) [reply src/test/java/com/SecureTrading/pageobjects/PaymentPage.java]: ok, I will change that :)

File: src/main/resources/__files/configSubmitCvvOnly.json (hunk 1 above)
Comment (pgs-gmartowski) [reply src/main/resources/__files/configSubmitCvvOnly.json]: @james-simpson-ST got it ;)

**Pull request merged**